HW2/05-form.js

```diff
@@ -23,5 +23,5 @@ function get_data(){
     }
```

Comment:
This file looks good honestly you might be able to add efficiency by make your console logs using string interpolation for example line 6 will turn from "console.log("Fullname: ", name);" to "console.log(`Fullname: ${name}`);". just keep in mind the "Fullname: ${name}" is wrapped in backtics `` git removes them for some reason when adding comments here.